src/cryptonote_core/blockchain.cpp

```
@@ -4549,6 +4951,75 @@ bool Blockchain::handle_block_to_main_chain(const block& bl, const crypto::hash&
+    }
+    catch (const std::exception& e)
+    {
+      MGINFO_RED("Failed to persist asset operation(s): " << e.what());
+      bvc.m_verifivation_failed = true;
+      return false;
+    }
+  }
+
+  // HF21: index every tx_out_zarcanum output in the per-asset LMDB table.
+  // This feeds the BGE surjection ring for future asset transfers.
+  if (get_current_blockchain_height() >= 1)
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)